app/mailers/contact_mailer.rb

```diff
@@ -1,17 +1,17 @@
 class ContactMailer < ActionMailer::Base
-  default from: "The Odin Project <admin@theodinproject.com>"
-  FEEDBACK_EMAIL = "feedback@theodinproject.com"
-  CONTACT_EMAIL = "contact@theodinproject.com"
-  ODIN_URL = "http://www.theodinproject.com"
+  default from: 'The Odin Project <admin@theodinproject.com>'
 
-
-  def suggestion_email(suggestion_text, url="", user_email="")
+  def suggestion_email(suggestion_text, url='', user_email='')
```

Comment:
I think we don't really need default values here. It'll better to change the `user_identifier` method as

```ruby
def user_identifier
  current_user_email || 'Anonymous'
end
```

What do you think?